picard/ui/options/general.py

```diff
@@ -17,8 +17,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
-from PyQt5 import QtCore
-from PyQt5.QtWidgets import QInputDialog
+from PyQt5 import QtCore,QtWidgets
```

Comment:
If you use `self.tagger.autoupdate_enabled` this likely isn't required.  If it is required, there should be a space after the comma.